sphinx/conf.py

```diff
@@ -77,7 +77,7 @@
 
 # mapping to other Sphinx projects
 # (allows us to reference objects from other projects)
-cylc_version = '8.0a2'
+cylc_version = '8.0b2'
```

Comment:
Will this be OK if testing with 8.0rc1?

Reply:
This is version used by intersphinx which is templated into the `cylc-doc` url for retrieving Sphinx objects.

This means we can do things like `:term:"cycle point"` to reference a term in the Cylc glossary.

The version used must be published so we will need to pin to the last release.